src/Status/StatusCountTable/StatusCountTable.tsx

```diff
@@ -0,0 +1,94 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Typography
+} from "@mui/material";
+
+import React from "react";
+import { Status } from "../statuses.types";
+import { StatusCountTableProps } from "./StatusCountTable.types";
+import StatusIcon from "../StatusIcon";
+
+// custom status count table component, which will be used in the status bar component on hover to show aditional data about statuses
+export function StatusCountTable({
+  title,
+  count,
+  tableSx = { borderRadius: "4px", boxShadow: 8, maxWidth: "280px" }
+}: StatusCountTableProps) {
+  const totalCount = Object.values(count).reduce((a, b) => a + b, 0);
+
+  /** Render every status name with first letter Capital and replace dashes with empty space */
+  function capitalizeFirstLetterAndRemoveDashes(str: string): string {
+    // Handle empty string case
+    if (!str) return str;
+    // Capitalize the first letter of a status
+    const capitalized = str.charAt(0).toUpperCase() + str.slice(1);
+    // replace dashes with empty space
+    if (capitalized.includes("-")) {
+      return capitalized.replace("-", " ");
+    }
+    return capitalized;
+  }
```

Comment:
We shouldn't need to do any manipulation like this. The renderable string for a given status is listed in the config. You should read it from there. https://github.com/IPG-Automotive-UK/react-ui/blob/be9eb6826ecd86132598aea5b96ea49f6e5128aa/src/Status/statuses.ts